src/snabbkaffe.erl

```diff
@@ -686,3 +679,13 @@ splitwith_(Pred, [Hd|Tail], Taken) ->
   end;
 splitwith_(_Pred, [], Taken) ->
   {lists:reverse(Taken), []}.
+
+mean([]) ->
+  0;
+mean([X|Rest]) ->
```

Comment:
`lists:sum(L) / length(L).`

Reply:
That's two list traversals instead of one.